wordpress-mobile/WordPress-iOS
# Add Jetpack powered overlay

Fixes #NA

It adds the "Jetpack powered" overlay and connects it to My Site home badge (for now)

Also adds support to connect any other banner/badge, and add a redirect action to the button in the overlay.
The image on top is temporary and will be replaced with an animation

**Known issues: the iPhone landscape design is still ongoing, and will be properly handled in a separate PR.**

<p align=center>
<img height=450 src=https://user-images.githubusercontent.com/34376330/181651131-fc2de33d-03c6-4807-8e0a-828faf9c18ba.png>
</p>

To test:
- Enable the `jetpackPoweredBottomSheet` feature flag
- Build/run this branch
- Go to My Site / Home and tap on the "Jetpack powered" badge
- Make sure the jetpack overlay appears (see screenshot above)
- Stop and disable the same flag
- Make sure the badge does not respond to touches and the overlay does not appear.

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
